velox/serializers/PrestoSerializer.cpp

```diff
@@ -294,6 +294,46 @@ void readValues<IntervalDayTime>(
   }
 }
 
+UnscaledLongDecimal readUnscaledLongDecimal(ByteStream* source) {
+  constexpr int64_t kInt64DeserializeMask = ~(static_cast<int64_t>(1) << 63);
```

Comment:
static constexpr int64_t

Reply:
The compiler will likely inline this variable. So I did not feel static was necessary.